tutorials/3d/standard_material_3d.rst

```diff
@@ -391,18 +288,18 @@ rim. If *Tint* is ``1``, then the albedo of the material is used. Using
 intermediate values generally works best.
 
 Clearcoat
-~~~~~~~~~
+---------
 
-*This feature is only available when using the GLES3 backend.*
+*This feature is only available when using the Vulkan backend.*
```

Comment:
I would remove these warnings for now. We can re add some if they turn out to be relevant